owl2nl/src/main/java/org/aksw/owl2nl/Optimizer.java

```diff
@@ -35,6 +35,8 @@ public String Optimise(String text) {
             System.out.println("Before optimize : " + text);
```

Comment:
Please change "Before Optimise" to "Text before our changes:"

Reply:
done